src/erldns_sup.erl

```diff
@@ -58,6 +58,7 @@ gc_registered(ProcessName) ->
 init(_Args) ->
     SysProcs =
         [
+            ?CHILD(pg, worker, [erldns]),
```

Comment:
`pg` means "process group"?

Reply:
Yes, OTP feature here https://www.erlang.org/doc/apps/kernel/pg.html, `segmented_cache` uses it.